src/client/src/ui/spotTile/components/SpotTile.tsx

```diff
@@ -46,23 +47,25 @@ export default class SpotTile extends PureComponent<Props, State> {
     const spotDate = priceData && spotDateFormatter(priceData.valueDate, false).toUpperCase()
 
     return (
-      <SpotTileWrapper className="spot-tile-container">
-        <SpotTileStyle className="spot-tile">
-          <Flex direction="column" justifyContent="space-between" height="100%">
-            <Flex alignItems="center" justifyContent="space-between">
-              <TileSymbol>{`${currencyPair.base}/${currencyPair.terms}`}</TileSymbol>
-              <DeliveryDate className="delivery-date">{spotDate && `SPT (${spotDate})`} </DeliveryDate>
+      <ThemeProvider theme={theme => theme.tile}>
```

Comment:
If we chose to use it like this we might consider a decorator. It might make some people more comfortable to think of it that way.

Reply:
Although — since that would increase the distance — I think it would just add indirection and confusion … as decorators usually do.